### PR TITLE
docs: update example for credential store

### PR DESCRIPTION
**Pull request by wangxiaoxuan273**

### CODE REVIEW

**Review by shizhMSFT — APPROVED** [review]
LGTM

File: docs/client_libraries/go.mdx [hunk 1]
```diff
@@ -10,7 +10,7 @@ Source code: [github.com/oras-project/oras-go](https://github.com/oras-project/o
 ## Introduction
 
 The ORAS Go client library provides the ability to replicate artifacts between different [Targets](./overview.mdx#target).  
-Furthermore, the version `v2` is a registry client conforming [image-spec v1.1.0-rc.5](https://github.com/opencontainers/image-spec/releases/tag/v1.1.0-rc5) and [distribution-spec v1.1.0-rc3](https://github.com/opencontainers/distribution-spec/releases/tag/v1.1.0-rc3).
+Furthermore, the version `v2` is a registry client conforming [image-spec v1.1.0-rc.6](https://github.com/opencontainers/image-spec/releases/tag/v1.1.0-rc6) and [distribution-spec v1.1.0-rc4](https://github.com/opencontainers/distribution-spec/releases/tag/v1.1.0-rc4).
```
Comment (shizhMSFT) [inline docs/client_libraries/go.mdx:13]: nit:
```suggestion
Furthermore, the version `v2` is a registry client conforming [image-spec v1.1.0-rc6](https://github.com/opencontainers/image-spec/releases/tag/v1.1.0-rc6) and [distribution-spec v1.1.0-rc4](https://github.com/opencontainers/distribution-spec/releases/tag/v1.1.0-rc4).

```

**Review by shizhMSFT — APPROVED** [review]
LGTM

**Review by Wwwsylvia — APPROVED** [review]
LGTM

**Pull request merged**